extensions/amp-scroll-toggle/amp-scroll-toggle.md

```diff
@@ -0,0 +1,82 @@
+<!--
+Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# `amp-scroll-toggle`
+
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>Slides a floating element in-and-out of view as the document is scrolled
+    up or down.</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Availability</strong></td>
+    <td><strong>Experimental</strong></td>
```

Comment:
@aghassemi Although I'm setting `experimental` here, it's only due to the rollout process. Unsure whether it's worth guarding this behind an actual flag.